### PR TITLE
ci: arm64 container follow-up

### PR DESCRIPTION
**Pull request by awelzel**

### CODE REVIEW

**Comment by awelzel** [comment]
I've ditched the arm_debian11_task due to actually causing test failures. Going to look at those separately.

File: docker/README [hunk 1]
```diff
@@ -1,19 +1,26 @@
 Container image for Zeek
 ========================
 
-This directory contains a minimal container image for Zeek. This image is
-published automatically to [DockerHub](https://hub.docker.com/u/zeekurity) for
+This directory allows building a minimal container image for Zeek. This image is
+published automatically to [DockerHub](https://hub.docker.com/u/zeek) for
```
Comment (timwoj) [inline docker/README:5]: Aren't we also publishing these to the AWS registry?

**Review by timwoj — APPROVED** [review]
Looks good. Just one question about the README.

File: docker/README (hunk 1 above)
Comment (awelzel) [reply docker/README:5]: Yes, I've added a small addition there.

**Pull request merged**